Mathlib/Analysis/CStarAlgebra/ApproximateUnit.lean

```diff
@@ -105,3 +112,235 @@ lemma CStarAlgebra.directedOn_nonneg_ball :
       have hab' : cfcₙ g a ≤ cfcₙ g a + cfcₙ g b := le_add_of_nonneg_right cfcₙ_nonneg_of_predicate
       exact CFC.monotoneOn_one_sub_one_add_inv cfcₙ_nonneg_of_predicate
         (cfcₙ_nonneg_of_predicate.trans hab') hab'
+
+section SpanNonneg
+
```

Comment:
Should this section really be in this file? It feels more general than approximate units.

Reply:
please suggest a better place. The reason it's here is that nowhere else imports `Analysis.SpecialFunctions.ContinuousFunctionalCalculus.PosPart`

Reply:
If you prefer, I could make a new file `Analysis.CStarAlgebra.SpecialFunctions.PosPart` and this could go in there (potentially along with some other things).

Reply:
@dupuisf FYI: I have made a new file.